src/components/ArticleList.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from "react";
+import axios from "axios";
+import { Placeholder, Grid } from "semantic-ui-react";
+
+export default class ArticleList extends Component {
```

Comment:
we can have a session turning this component into a functional component with react hooks tomorrow